modules/olm-metrics.adoc

```diff
@@ -13,15 +13,33 @@ use by the Prometheus-based {product-title} cluster monitoring stack.
 |===
 |Name |Description
 
+|`catalog_source_count`
+|Number of CatalogSources.
+
+|`csv_abnormal`
+|When reconciling a CSV, present whenever a CSV is in a failed state, per
+version. Includes the `name`, `namespace`, `phase`, `reason`, and `version`
```

Comment:
The "per version" here and in `csv_succeeded` is a bit confusing to me. But I'm not sure how to clarify it.

Reply:
Changing to "When reconciling a CSV, present whenever a CSV version is in a failed state."